config-examples/node.yaml

```
@@ -36,17 +53,17 @@ open_blobs_hard_limit: 10
 http_api_port: 8000
 # http REST api address
 http_api_address: 0.0.0.0
-# memory limit for all bloom filters. Unlimited if not specified
-bloom_filter_memory_limit: 8589934592
+# [size] memory limit for all bloom filters. Unlimited if not specified
+bloom_filter_memory_limit: 8589934592 b
```

Please, make this value esier to read by converting to Gb:
```suggestion
bloom_filter_memory_limit: 8 gb
```